feeds/people2.json

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
Migrate usage of `feeds/people.json` to this to allow rename to `people.json`. Think it's just the bio collect form.